[2021-07-02 05:40:38,129] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-09-28T05:55:00+00:00 [queued]>
[2021-07-02 05:40:38,150] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-09-28T05:55:00+00:00 [queued]>
[2021-07-02 05:40:38,150] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:40:38,150] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-07-02 05:40:38,151] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:40:38,172] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): print_date> on 2020-09-28T05:55:00+00:00
[2021-07-02 05:40:38,181] {standard_task_runner.py:52} INFO - Started process 159561 to run task
[2021-07-02 05:40:38,194] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'print_date', '2020-09-28T05:55:00+00:00', '--job-id', '727', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpvc3c36f_', '--error-file', '/tmp/tmpjraexvgd']
[2021-07-02 05:40:38,212] {standard_task_runner.py:77} INFO - Job 727: Subtask print_date
[2021-07-02 05:40:38,338] {logging_mixin.py:104} INFO - Running <TaskInstance: test.print_date 2020-09-28T05:55:00+00:00 [running]> on host 9b3ed935c8bc
[2021-07-02 05:40:38,471] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@airflow.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2020-09-28T05:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-28T05:55:00+00:00
[2021-07-02 05:40:38,472] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-02 05:40:38,472] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'date']
[2021-07-02 05:40:38,493] {subprocess.py:75} INFO - Output:
[2021-07-02 05:40:38,503] {subprocess.py:79} INFO - Fri Jul  2 05:40:38 UTC 2021
[2021-07-02 05:40:38,504] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-02 05:40:38,603] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test, task_id=print_date, execution_date=20200928T055500, start_date=20210702T054038, end_date=20210702T054038
[2021-07-02 05:40:38,695] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-07-02 05:40:38,749] {local_task_job.py:151} INFO - Task exited with return code 0
